### DISCUSSION TITLE
Autostart not launching autokey on startup

### DISCUSSION THREAD

**Discussion by Momijie** [autokey/autokey#729] [Q&A]
I enable the 'Automatically start AutoKey at login', but it doesn't seem to be doing anything beside putting a .desktop file in autostart.
Running on an up to date arch installation with lightdm launching into a i3wm instance.

**Comment by josephj11** [comment]
I'm not familiar with Arch.

I believe our lead developer is familiar with Arch, but until someone (like him) answers  this:

I looked [here](https://bbs.archlinux.org/viewtopic.php?id=262771) and it seems to say that the desktop file should be all that's needed.

    There's always the 'autostart' way, if you create a dir. like '~/.config/autostart' and place the F.I. 'netctl-tray-auto.desktop' in it, it will be started.

Does AutoKey run correctly when you start it manually from your start menu or from the CLI?

I also found [this](https://i3wm.org/docs/userguide.html#_automatically_starting_applications_on_i3_startup) which includes

    4.19. Automatically starting applications on i3 startup

**Comment by Elliria** [comment]
Also, AutoKey may actually be running. Besides not seeming to do anything, does it also fail to respond when you use one of its triggers?

AutoKey will put itself into the system tray by default without opening a window and will sit there waiting for you to use a trigger and then it will do its thing. If you'd like it to open its window when it launches, you can add the **-c** option to its executable command. For instance:

`autokey-gtk -c`

or:

`autokey-qt -c`

Both of my shortcuts use the **-c** option.

**Reply by josephj11** [reply]
i forgot about that.
